cpp/shared/most_common.h

```diff
@@ -28,6 +32,26 @@ double most_common(byte* data, const long len, const int alph_size, const int ve
 		printf("%s Most Common Value Estimate: p_u = %.17g\n", label, ubound);
 		printf("%s Most Common Value Estimate: min entropy = %.17g\n", label, entEst);
 	}
-
+        string literal = "Literal";
+        
+        if(literal.compare(label)) {
+            tc.SetLiteral_mcv_estimate_mode(mode);
+            tc.SetLiteral_mcv_estimate_p_hat(pmax);
+            tc.SetLiteral_mcv_estimate_p_u(ubound);
+        } else {
+            tc.SetBitstring_mcv_estimate_mode(mode);
+            tc.SetBitstring_mcv_estimate_p_hat(pmax);
+            tc.SetBitstring_mcv_estimate_p_u(ubound);
+        }
+        
 	return entEst;
 }
+
+// Wrapper method needed because some runs do not get output as JSON currently
+// and therefore do not have a TestCase object to send
+double most_common(byte* data, const long len, const int alph_size, const int verbose, const char *label){

```

Comment:
This isn't clear. Why is this overloading being done rather than just changing the way the function works? These functions aren't called from a lot of places, so there's not much benefit to this extra layer of indirection. in this one case.